src/command_modules/azure-cli-servicebus/HISTORY.rst

```diff
@@ -4,8 +4,12 @@ Release History
 ===============
 
 0.2.1
-++++++
-* Minor fixes.
++++++
+* Added migration command group to migrate a namespace from Service Bus Standard to Premium
+
+* Added new optional properties to Service Bus queue and Subscription
+    --enable-batched-operations and --enable-dead-lettering-on-message-expiration in queue
+    --dead-lettering-filter-evaluation-exceptions in subscriptions
```

Comment:
These are extremely long parameter names. This makes for a less than ideal experience for customers. 
I see you've talked to @tjprescott about this, if there is any way to use something shorter, I'd second his recommendation to do so.

Reply:
changed to '--dead-letter-on-filter-exceptions'